### DISCUSSION TITLE
表格复选框用table.checkStatus()获取的数据没有行索引 INDEX，请问如何得到对应行索引？

### DISCUSSION THREAD

**Discussion by zzlatm** [layui/layui#1858] [问答]
### 议题条件

- [X] 我确认已查看官方使用文档：https://layui.dev ，但没有找到相关解决方案。
- [X] 我确认已在 [Issues](https://github.com/layui/layui/issues?q=is%3Aissue ) 中搜索过类似的问题，但没有找到相关解决方案。
- [X] 我已仔细阅读： 🍀[ Layui Issue 贡献指南](https://github.com/layui/layui/blob/main/CONTRIBUTING.md)

### 议题类型

功能请求

### 使用版本

v2.9.8

### 问题描述

table.checkStatus()方法得到的只有纯数据，没有行索引（INDEX)，请问如何获得，我需要配合updateRow方法更新行数据

### 业务代码

```auto
var tableStatus = table.checkStatus('test');
console.log(tableStatus.data) // 选中行的数据
```

### 浏览器

EDGE

### 友好承诺

- [X] 我承诺将本着相互尊重、理解和友善的态度进行交流，共同维护 Layui 良好的社区氛围。

**Comment by sentsim** [comment]
见文档“返回数据中的特定字段”：https://layui.dev/docs/2/table/#data.reserved
